### ISSUE TITLE
Add titles to icons

### ISSUE THREAD

**Issue by LeandraH** [digitalfabrik/integreat-app#2470]
**Is your feature request related to a problem? Please describe.**
We should check if some of our icons have more than a purely decorative purpose and if they do, give them a [title](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title) or an [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby).

These titles/labels should be added to our translations.json and the `t` function from `react-i18next` should be used to translate them in the code.

**Comment by LeandraH** [comment]
I think we haven't worked on this ticket in more than a year because it's not very well defined. If we come across an icon that needs a title, we can open a ticket for that icon.
